services/QuillLMS/bin/rspec

```diff
@@ -2,9 +2,9 @@
 # frozen_string_literal: true
 
 begin
-  load File.expand_path("../spring", __FILE__)
-rescue LoadError
-  nil
+  load File.expand_path('../spring', __FILE__)
+rescue LoadError => e
+  raise unless e.message.include?('spring')
```

Comment:
Is this because the error message will include the string 'spring' when rails is in develop mode, but not production?

Reply:
So this is the generated code written by the [Spring config script](https://github.com/rails/spring#setup). I believe the intention is to swallow errors related to loading spring (spring is additive for caching purpose, but doesn't affect the code, so we shouldn't error for it), but not other non-spring errors.

Note, this is an `rspec` file, so it's not related to development or production.